src/main/java/uk/gov/justice/hmpps/casenotes/sync/SyncCaseNotes.kt

```diff
@@ -97,6 +97,16 @@ class SyncCaseNotes(
     }
   }
 
+  fun moveCaseNotes(request: MoveCaseNotesRequest) {
+    val from = noteRepository.findAllByIdIn(request.caseNoteIds)
+      .filter { it.personIdentifier == request.fromPersonIdentifier }
```

Comment:
Thinking the correct check is that all the supplied notes are associated either with the from or the to person identifier. Anything else would be a 400 indicating that the two services aren't in sync?